st2auth/st2auth/app.py

```diff
@@ -65,9 +65,9 @@ def setup_app(config={}):
 
     app = router.as_wsgi
 
+    app = ErrorHandlingMiddleware(app)
```

Comment:
Yeah, adding what and why would be great.

And yes, having it in 3 places in slightly redundant, but this could also mean code could be refactored (if possible).

In any case, I still think it would be good to add it because without a comment (and tests) it's just a question of time until someone starts moving things around when refactoring it or similar...